project.clj

```diff
@@ -104,7 +105,8 @@
                                      "test/resources/hooks/hook-example-0.1.0-SNAPSHOT.jar"]}
              :prepush {:plugins [[yogsototh/lein-kibit "0.1.6-SNAPSHOT"]
                                  [lein-bikeshed "0.3.0"]]}}
-  :plugins [[lein-shell "0.5.0"]]
+  :plugins [[lein-shell "0.5.0"]
+            [perforate "0.3.4"]]
```

Comment:
Do we want to consider setting up:
`:perforate { :environments [{:name :default
                                             :namespaces [ctia.http.routes.actor-bench]`
This sets up a template to make expansion of the benchmarking environment easier in the future (if there is the possibility of expanding it).